covid_moonshot_ml/tests/test_covid_moonshot_ml.py

```diff
@@ -5,10 +5,6 @@
 # Import package, test suite, and other packages as needed
 import sys
 
-import pytest
```

Comment:
I think we actually want to keep these since this will be a unit test in the future but since it is so undeveloped now I don't think its worth haggling about, instead I made this issue: https://github.com/choderalab/covid-moonshot-ml/issues/53